article/_posts/2020-10-15-vision_transformer.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: review
+title: "An Image is Worth 16x16 Words: Transformers for Image Recognition at Scale"
+tags: cnn transformer classification
+author: "Carl Lemaire"
+cite:
+    authors: "Anonymous (Google?)"
```

Comment:
I would not imagine about the authors. Even more so if this is a double blind review. If approved, we can submit a separate PR to honor the authors.